src/Pages/ProductList/ProductSpec/Review/Review.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+
+import ReviewModal from './ReviewModal/ReviewModal';
+import { API, Token } from '../../../../config';
+
+import './Review.scss';
+
+const Review = () => {
+  const [isClicked, setIsClicked] = useState(false);
+  const [reviewList, setReviewList] = useState([]);
+
+  const getData = async () => {
+    const data = await (await fetch(`${API.products}/1`)).json();
```

Comment:
```suggestion
    const data = await fetch(`${API.products}/1`).then(res => res.json());
```
이부분 래영님이 리뷰해준 부분이네요! 
그리고 ${API.products}/1`에서 1의 id 부분도 동적으로 바껴야 할 것 같네요:)

Reply:
네 수정하였습니다!